scripts/generate_golang.sh

```diff
@@ -29,4 +19,5 @@ mock_api_file="$OUTPUT_PATH/default_api_mock.go"
 rm -rf $mock_api_file
 moq -out "$mock_api_file" "$OUTPUT_PATH" DefaultApi
 
+cp $OPENAPI_FILENAME .openapi/$OPENAPI_FILENAME
```

Comment:
What is the main motivation for retaining the OpenAPI files at the root instead of in each generated client package? What happens when the name is not unique? We should consider renaming it to the name of the service or something.

Reply:
I agree - let's introduce id's to be sent from source repositories along with the filenames and use those id's instead of file names.